NoComponentDrag.py

```diff
@@ -101,8 +94,7 @@ def get_direct_edit_drag_enabled():
     return fusion_drag_controls_cmd_def_.controlDefinition.isChecked
 
 def check_environment():
-    global enable_cmd_def_
-    global parametric_environment_
+    global enable_cmd_def_, parametric_environment_
```

Comment:
Condensed this since the global keyword implies exposing variables and is therefore not needed to be broken into multiple lines